components/items/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+import { ProductProps } from "@/types/product";
+import styles from "./Card.module.css";
+import emptyImg from "@/public/img_default.svg";
+
+const Card = ({ product }: { product: ProductProps }) => {
```

Comment:
P3:
product 객체로 받을 필요가 없어 보입니다.

```suggestion
const Card = ({ id, name, images, price, favoriteCount }: ProductProps) => {
```